<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>496</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Anlässe und Gruppen bearbeiten</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Einträge, für die es schon Notentabellen gibt, dürfen nicht geändert oder gelöscht werden.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Anlässe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="occasion_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>200</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>1. Halbjahr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2. Halbjahr</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Abitur</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="occasion_editor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QToolButton" name="pb_remove_occasion">
           <property name="toolTip">
            <string>Den aktuellen Anlass löschen.</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_remove.svg</normaloff>../icons/icon_remove.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pb_new_occasion">
           <property name="toolTip">
            <string>Den neu eingegebenen Anlass in die Liste aufnehmen.</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_new.svg</normaloff>../icons/icon_new.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="pb_edited_occasion">
           <property name="toolTip">
            <string>Den aktuellen Anlass umbenennen.</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_ok.svg</normaloff>../icons/icon_ok.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Klassen/Gruppen</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="group_list">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>0</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <item>
          <property name="text">
           <string>13</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>12G.G</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QComboBox" name="combo_classes">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>13</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>12G.G</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="group_editor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="pb_remove_group">
           <property name="toolTip">
            <string>Aktuelle Klasse/Gruppe (für den aktuellen Anlass) löschen.</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_remove.svg</normaloff>../icons/icon_remove.svg</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pb_new_group">
           <property name="toolTip">
            <string>Die neu angegebene Klasse/Gruppe in die Liste aufnehmen.</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_new.svg</normaloff>../icons/icon_new.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableWidget" name="report_table">
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>0</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row>
          <property name="text">
           <string>New Row</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Zeugnis-Art</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Vorlage</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>Zeugnis</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>Zeugnis-Vorlage_1</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="pb_remove_report">
           <property name="toolTip">
            <string>Aktuelle Zeugnis-Art (für aktuellen Anlass und aktuelle Gruppe) löschen.</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../icons/icon_remove.svg</normaloff>../icons/icon_remove.svg</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>occasion_list</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>occasion_editor</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>62</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
